apps/src/templates/MultipleSectionsAssigner.jsx

```diff
@@ -20,7 +20,7 @@ const MultipleSectionsAssigner = ({
   courseOfferingId,
   courseVersionId,
   scriptId,
-  reassignConfirm,
+  reassignConfirm = () => {},
```

Comment:
Will implement a custom reassignConfirm as part of this ticket https://codedotorg.atlassian.net/browse/ACQ-643

Reply:
Thank you for tracking all the followup work so neatly!